File: Java/2114_Maximum_Number_of_Words_Found_in_Sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int result = Integer.MIN_VALUE;
        for(int i = 0; i < sentences.length; i++){
            String[] words = sentences[i].split(" ");
            result = Math.max(result, words.length);
        }
        return result;
    }
}

File: Python3/2114_Maximum_Number_of_Words_Found_in_Sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        return max([sen.count(" ") for sen in sentences]) + 1
        """
        result = 0
        for sen in sentences:
            t = sen.count(" ") + 1
            result = max([t, result])
        return result
        """
